packages/ag-charts-community/src/chart/series/series.ts

```diff
@@ -645,12 +645,10 @@ export abstract class Series<
     }
 
     protected pickNodeExactShape(point: Point): SeriesNodePickMatch | undefined {
-        const match = this.contentGroup.pickNode(point.x, point.y);
-        if (match && match.datum.missing !== true) {
-            return { datum: match.datum, distance: 0 };
+        const datum = this.contentGroup.pickNode(point.x, point.y)?.closestDatum();
```

Comment:
Nit: Could we extract these changes out distinctly as they aren't related to validation changes?